src/mock/vitest-spy.ts

```diff
@@ -0,0 +1,580 @@
+// TODO: use @vitest/spy when migrate to esm
```

Comment:
This file copied as is from @vitest/spy and used in nodejs environment. In browser environment this file replaced to special file which works only in browser env.

Copied because module is esm and I can't use it correctly in cjs world.